lib/backend/octokit.ts

```diff
@@ -1,32 +1,66 @@
 import { retry } from '@octokit/plugin-retry';
 import { throttling } from '@octokit/plugin-throttling';
 import { Octokit } from '@octokit/rest';
+import { knuthShuffle } from 'knuth-shuffle';
 
-const auth = process.env.PLN_ADMIN_GITHUB_TOKEN;
-if (!auth) {
+
+const authTokens = new Map<string, boolean>();
+const potentialTokens = [
+  process.env.PLN_ADMIN_GITHUB_TOKEN,
+  process.env.PLN_ADMIN_GITHUB_TOKEN2,
+  process.env.PLN_ADMIN_GITHUB_TOKEN3,
+];
```

Comment:
could also update this to support comma-delimited tokens so we don't need to keep modifying the code, but I was trying to get this fix out fast